.github/workflows/register_repos.yaml

```diff
@@ -0,0 +1,54 @@
+name: Register repository
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  register-repository:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Register new repository
+        env:
+          GH_USERNAME: ${{ secrets.GH_USERNAME }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          python -m pip install requests ruamel.yaml aiohttp
+          python .github/workflows/register_repositories.py
+
+      - name: Capture diff output
+        id: diff
+        shell: bash
+        run: |
+          git diff config.yaml > config.diff
+
+      - name: Pushing up-to-date config file
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "GitHub Action"
+          git add config.yaml && git commit -m "[skip ci] Update configuration file."
+          git push -f --set-upstream origin master
+
+      - name: Send a reply message to the issue
+        uses: actions/github-script@0.9.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs')
+            const diff = fs.readFileSync('config.diff', 'utf8')
+            const logs = fs.readFileSync('hooks_log.txt', 'utf8')
+            const issue_reply = "👋 the following changes were made:\n\n" + "```bash\n" + `${diff}` + "```\n" + `${logs}`+ "\n=== Check the master branch for latest changes. ==="
```

Comment:
Can we format the last line of this response with markdown?  For example, instead of:

`"=== Check the master branch for latest changes. ==="`

Can we italicize it?

`"*Check the master branch for latest changes.*"`

or bold+italicize?

`"***Check the master branch for latest changes.***"`